fiftyone/utils/sam3.py

```
@@ -0,0 +1,537 @@
+        """
+        # TODO: FIX THIS! Using sam3 predict_inst_batch
+        images = imgs.pop("image")
+
+        self._sam_predictor.processor.set_image_batch(images)
+
+        point_coords = imgs.get("point_coords")
+        point_labels = imgs.get("point_labels")
+        boxes = imgs.get("boxes")
+        mask_inputs = imgs.get("mask_inputs")  # Not used currently
+        multimask_output = (
+            True if (boxes is None and point_coords is not None) else False
```

Ruff (0.15.11)

[warning] Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)